gcloud-java-resourcemanager/src/main/java/com/google/gcloud/resourcemanager/ResourceManager.java

```diff
@@ -366,15 +336,20 @@ public String value() {
    * @see <a href=
    *     "https://cloud.google.com/resource-manager/reference/rest/v1beta1/projects/testIamPermissions">
    *     Resource Manager testIamPermissions</a>
+   * @see <a href=
+   *     "https://cloud.google.com/iam/#supported_cloud_platform_services">Supported Cloud Platform
+   *     Services</a>
    */
-  List<Boolean> testPermissions(String projectId, List<Permission> permissions);
+  List<Boolean> testPermissions(String projectId, List<String> permissions);
```

Comment:
per our earlier discussion. I am fine with having just one flavour of this method.